README.md

```diff
@@ -0,0 +1,25 @@
+# hellomicroverse
+# Project Name
+
+> Hello World Microverse Project.
+
+
```

Comment:
- [ ] Kindly add more description about your project, this is usually the first point of interaction between your project and the world, showcasing what the project is all about and how to use it..

Reply:
Thank you for the review. I have added the pull request comments